дз 6/дз 6/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+
+namespace дз_6
+{
+    class Program
+    {
+        static int method(int n)
```

Comment:
Принято имя метода с большой буквы
```suggestion
        static int Method(int n)
```